cmd/podman/common/create.go

```diff
@@ -393,6 +393,11 @@ func GetCreateFlags(cf *ContainerCLIOpts) *pflag.FlagSet {
 		"rootfs", false,
 		"The first argument is not an image but the rootfs to the exploded container",
 	)
+	createFlags.StringVar(
+		&cf.SdNotifyMode,
+		"sdnotify", "container",
+		`control sd-notify behavior ("container"|"conmon-only"|"ignore")`,
```

Comment:
Change conmon-only to conmon.  The -only does not add any value, and I am not a big fan of options with "dashes" in them.